src/lyyti/events.py

```diff
@@ -71,9 +72,29 @@ def parse_custom_field(custom: dict[str, dict[str, str]]) -> Custom:
     return custom_fields
 
 
+def has_ended(unix_time_utc: int) -> bool:
+    """
+    Checks whether the given Unix UTC timestamp is in the past,
+    relative to the start of the current day (12:00 am UTC), exclusive.
+
+    Args:
+        unix_time (int): Unix timestamp to check
+
+    Returns:
+        bool: True if the timestamp is in the future, False otherwise
+    """
+    start_of_day_utc = datetime.utcnow().replace(
+        hour=0, minute=0, second=0, microsecond=0
+    )
+    start_of_day_unix = int(start_of_day_utc.timestamp())
+    return unix_time_utc > start_of_day_unix
```

Comment:
Why hours, minutes, seconds, and microseconds are excluded? Consider this situation: Current time is 14.00 and course is ended already at 12.00. Current version will return false instead of true. @anttipham, please check this.

Reply:
That is not a problem. It's a feature. (Said every programmer ever)

Customer said: "You can just use day level scale. If Lyyti event ended before today, it’s safe to ignore it."
So I'm just comparing the time to 12 am (klo 0.00).